bundle/metadata/annotations.yaml

```diff
@@ -1,11 +1,10 @@
 annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: alpha
   operators.operatorframework.io.bundle.channels.v1: alpha
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.mediatype.v1: registry+v1
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: antrea-operator-for-kubernetes
-  operators.operatorframework.io.metrics.builder: operator-sdk-v1.2.0
+  operators.operatorframework.io.metrics.builder: operator-sdk-v0.19.0
```

Comment:
Could you use operator-sdk 1.2+ for generating.

Reply:
Thanks, I'll update the version.